### DISCUSSION TITLE
This error appears when I bind a domain name.

### DISCUSSION THREAD

**Discussion by dllio** [community/community#73444] [New to GitHub]
This error appears when I bind a domain name：
 The custom domain `***.***` is already taken. If you are the owner of this domain, check out https://docs.github.com/pages/configuring-a-custom-domain-for-your-github-pages-site/verifying-your-custom-domain-for-github-pages for information about how to verify and release this domain.

**Comment by Seol-JY** [comment]
If you are verifying a domain you own, which is currently in use by another user or organization, to make it available for your GitHub Pages website; note that the process to release the domain from its current location will take 7 days to complete. If you are attempting to verify an already verified domain (verified by another user or organization), the release process will not be successful.

**Reply by dllio** [reply]
Thank you very much~
I have verified the ownership of the domain name and added relevant configurations to the DNS settings of my personal domain name.
I'll wait another 7 days and observe again.

**Reply by UtterDonkey** [reply]
I'm having the same problem but the domain name has never been used anywhere else.
